server/graphql_schemas/event/schema.py

```diff
@@ -98,6 +100,7 @@ def mutate_and_get_payload(cls, root, info, **input):
             )
             new_event = CreateEvent.create_event(
                 category, user_profile, **input)
+            add_event_to_calendar(user_profile, new_event)
```

Comment:
Great stuff @njeri-ngigi. Can we make this an async background task (using the BackgroundTaskWorker) so that the user doesn't have to wait for the Calendar API call to return before they get a response.

Reply:
all done